feat(android): add Android.{launchServer,connect} (#18263)
microsoft/playwright: docs/src/api/class-android.md

PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 npm i -D playwright
```

## async method: Android.connect
* since: v1.28
- returns: <[AndroidDevice]>

> This methods attaches Playwright to an existing Android instance.

Reviewer: ```suggestion
This methods attaches Playwright to an existing Android device.
```

Prevents automatic playwright driver installation on attach. Assumes that the drivers have been installed already.

## async method: Android.launchServer
* since: v1.28
* langs: js
- returns: <[BrowserServer]>

> Returns the Android browser instance. Launches Playwright Android server that client can connect to. See the following example:

Reviewer: ```suggestion
Launches Playwright Android server that clients can connect to. See the following example:
```

  await page.goto('https://webkit.org/');
  console.log(await page.evaluate(() => window.location.href));
  await page.screenshot({ path: 'page-chrome-1.png' });

  await context.close();
})();
```

### option: Android.launchServer.adbHost
* since: v1.28
- `adbHost` <[string]>

Optional host to establish ADB server connection. Default to `127.0.0.1`.

### option: Android.launchServer.adbPort
* since: v1.28
- `adbPort` <[int]>

Optional port to establish ADB server connection. Default to `5037`.

### option: Android.launchServer.omitDriverInstall
* since: v1.28
- `omitDriverInstall` <[boolean]>

Prevents automatic playwright driver installation on attach. Assumes that the drivers have been installed already.

### option: Android.launchServer.deviceSerialNumber
* since: v1.28
- `deviceSerialNumber` <[string]>

Optional device serial number to launch the browser on. If not specified, it will
throw if multiple devices are connected.

### option: Android.launchServer.port
* since: v1.28
- `port` <[int]>

Port to use for the web socket. Defaults to 0 that picks any available port.

### option: Android.launchServer.wsPath
* since: v1.28
- `wsPath` <[string]>

> Path at which to serve the Browser Server. For security, this defaults to an

Reviewer: ```suggestion
Path at which to serve the Android Server. For security, this defaults to an
```
